### PR TITLE
full scheduler v2 CI testing

### PR DESCRIPTION
**Pull request by jrhizor**
Now that we'll be running scheduler v2 in cloud, we need to have regression tests to prevent regressions like https://github.com/airbytehq/airbyte/pull/10198

### CODE REVIEW

File: airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/AcceptanceTests.java [hunk 1]
```diff
@@ -721,8 +721,6 @@ public void testIncrementalDedupeSync() throws Exception {
 
   @Test
   @Order(14)
-  @DisabledIfEnvironmentVariable(named = "KUBE",
```
Comment (jrhizor) [inline airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/AcceptanceTests.java:724]: We should be testing at least one sync failure case.

**Comment by jrhizor** [comment]
@benmoriceau I think we're going to need to work together to get these running reliably locally at least.

Right now I'm running into reset errors and cancellation errors.

Reset error example:
https://scans.gradle.com/s/5vj5gg57cd5qy/tests/:airbyte-tests:acceptanceTests/io.airbyte.test.acceptance.AcceptanceTests/testIncrementalSync()?top-execution=1

Cancellation error example:
```
Caused by: io.grpc.StatusRuntimeException: INVALID_ARGUMENT: java.lang.IllegalArgumentException: Unknown query type: getState, knownTypes=[]
	at io.temporal.internal.sync.QueryDispatcher.handleQuery(QueryDispatcher.java:79)
	at io.temporal.internal.sync.SyncWorkflowContext.handleQuery(SyncWorkflowContext.java:245)
	at io.temporal.internal.sync.WorkflowExecuteRunnable.handleQuery(WorkflowExecuteRunnable.java:121)
	at io.temporal.internal.sync.SyncWorkflow.query(SyncWorkflow.java:187)
	at io.temporal.internal.replay.ReplayWorkflowExecutor.query(ReplayWorkflowExecutor.java:136)
	at io.temporal.internal.replay.ReplayWorkflowRunTaskHandler.handleQueryWorkflowTask(ReplayWorkflowRunTaskHandler.java:245)
	at io.temporal.internal.replay.ReplayWorkflowTaskHandler.handleWorkflowTaskWithQuery(ReplayWorkflowTaskHandler.java:117)
	at io.temporal.internal.replay.ReplayWorkflowTaskHandler.handleWorkflowTask(ReplayWorkflowTaskHandler.java:97)
	at io.temporal.internal.worker.WorkflowWorker$TaskHandlerImpl.handle(WorkflowWorker.java:237)
	at io.temporal.internal.worker.WorkflowWorker$TaskHandlerImpl.handle(WorkflowWorker.java:195)
	at io.temporal.internal.worker.PollTaskExecutor.lambda$process$0(PollTaskExecutor.java:73)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)

	at io.grpc.stub.ClientCalls.toStatusRuntimeException(ClientCalls.java:262) ~[grpc-stub-1.42.1.jar:1.42.1]
	at io.grpc.stub.ClientCalls.getUnchecked(ClientCalls.java:243) ~[grpc-stub-1.42.1.jar:1.42.1]
	at io.grpc.stub.ClientCalls.blockingUnaryCall(ClientCalls.java:156) ~[grpc-stub-1.42.1.jar:1.42.1]
	at io.temporal.api.workflowservice.v1.WorkflowServiceGrpc$WorkflowServiceBlockingStub.queryWorkflow(WorkflowServiceGrpc.java:2989) ~[temporal-serviceclient-1.6.0.jar:?]
	at io.temporal.internal.external.GenericWorkflowClientExternalImpl.lambda$query$5(GenericWorkflowClientExternalImpl.java:220) ~[temporal-sdk-1.6.0.jar:?]
	at io.temporal.internal.retryer.GrpcSyncRetryer.retry(GrpcSyncRetryer.java:61) ~[temporal-serviceclient-1.6.0.jar:?]
	at io.temporal.internal.retryer.GrpcRetryer.retryWithResult(GrpcRetryer.java:51) ~[temporal-serviceclient-1.6.0.jar:?]
	at io.temporal.internal.external.GenericWorkflowClientExternalImpl.query(GenericWorkflowClientExternalImpl.java:213) ~[temporal-sdk-1.6.0.jar:?]
	at io.temporal.internal.client.RootWorkflowClientInvoker.query(RootWorkflowClientInvoker.java:141) ~[temporal-sdk-1.6.0.jar:?]
	at io.temporal.internal.sync.WorkflowStubImpl.query(WorkflowStubImpl.java:342) ~[temporal-sdk-1.6.0.jar:?]
	... 62 more
```

To run the tests I'm running `AcceptanceTests` from IntelliJ with the env var configuration `KUBE=true;USE_EXTERNAL_DEPLOYMENT=true;NEW_SCHEDULER=true` against a local Kubernetes instance started with `kubectl apply -k kube/overlays/dev-integration-test-schedulerv2`.

None of these problems look related to the container orchestrator so I'm going to just continue with that and leave this PR for us to iterate on.

**Review by pmossman — APPROVED** [review]
I think this all looks sensible to me. Build times are going to effectively double, right? Unfortunate but seems necessary if we want solid coverage while we're in between the old and new

**Comment by jrhizor** [comment]
It'll be in parallel so not double, just more expensive. Still not ideal though.

**Pull request merged**